Description: Bug fix of comparison function
 See module/recognizer/tomoe-recognizer-simple-logic.c
Author: NIIBE Yutaka <gniibe@fsij.org>
Forwarded: Reported, but this workaround patch is not forwarded
Reviewed-By: NIIBE Yutaka <gniibe@fsij.org>
Last-Update: 2010-05-12

--- tomoe-0.6.0.orig/lib/tomoe-context.c	2007-04-19 13:10:31.000000000 +0900
+++ tomoe-0.6.0/lib/tomoe-context.c	2010-05-12 14:38:51.000000000 +0900
@@ -293,9 +293,7 @@
 static gint
 _candidate_compare_func (gconstpointer a, gconstpointer b)
 {
-    TomoeCandidate *ca = *(TomoeCandidate **) a;
-    TomoeCandidate *cb = *(TomoeCandidate **) b;
-    return tomoe_candidate_compare (ca, cb);
+    return tomoe_candidate_compare (a, b);
 }
 
 
